ethereum/dbstack/db_stack.go

```diff
@@ -120,4 +120,4 @@ func (d *DBStack) Checkpoint() (int, error) {
 		return 0, err
 	}
 	return d.simpleStack.Checkpoint()
-}
+}
```

Comment:
nit: Can you add a new line to the end of this file? Your editor should be configured to do this automatically. The new line was there before and it looks like you removed it.